examples/sps/multiapp/yttria_thermoelectric/reference_residual/function_electricpotential/micro_yttria_thermoelectric_oneway.i

```diff
@@ -296,79 +298,79 @@ initial_field=10 #from the engineering scale, starting value 10 V/m
     constant_expressions = '15              5.52e-2' #eps_void_over_e in 1/V/nm
     derivative_order = 2
     expression = '-hs * eps_rel_solid * eps_void_over_e - hv * eps_void_over_e'
-  [../]
-  [./solid_pre]
+  []
+  [solid_pre]
     type = DerivativeParsedMaterial
     property_name = solid_pre
     material_property_names = 'hs ns_y_min ns_o_min'
     constant_names =       'Z_cat   Z_an'
     constant_expressions = '-3      2'
     derivative_order = 2
     expression = '-hs * (Z_cat * ns_y_min + Z_an * ns_o_min)'
-  [../]
-  [./void_pre]
+  []
+  [void_pre]
     type = DerivativeParsedMaterial
     property_name = void_pre
     material_property_names = 'hv'
     constant_names =       'Z_cat   Z_an nv_y_min nv_o_min'
     constant_expressions = '-3      2    26.837   40.2555'
     derivative_order = 2
     expression = '-hv * (Z_cat * nv_y_min + Z_an * nv_o_min)'
-  [../]
-  [./cat_mu_pre]
+  []
+  [cat_mu_pre]
     type = DerivativeParsedMaterial
     property_name = cat_mu_pre
     material_property_names = 'hs hv ks_cat kv_cat' #TODO add material properties for these
     constant_names =       'Z_cat'
     constant_expressions = '-3'
     derivative_order = 2
     expression = '-hs * Z_cat / ks_cat - hv * Z_cat / kv_cat'
-  [../]
-  [./an_mu_pre]
+  []
+  [an_mu_pre]
     type = DerivativeParsedMaterial
     property_name = an_mu_pre
     material_property_names = 'hs hv ks_an kv_an' #TODO add material properties for these
     constant_names =       'Z_an'
     constant_expressions = '2'
     derivative_order = 2
     expression = '-hs * Z_an / ks_an - hv * Z_an / kv_an'
-  [../]
-  [./cat_V_pre]
+  []
+  [cat_V_pre]
     type = DerivativeParsedMaterial
     property_name = cat_V_pre
     material_property_names = 'hs hv ks_cat kv_cat' #TODO add material properties for these
-    constant_names =       'Z_cat   v_scale e '
+    constant_names = 'Z_cat   v_scale e '
```

Comment:
Remember that the names and expression values (next line down) need to be vertically aligned. Please add the space in front of the `Z_cat` name back to maintain the vertical alignment